crates/memory/src/op_store/time_slice_hash_store.rs

```diff
@@ -0,0 +1,532 @@
+use kitsune2_api::{K2Error, K2Result};
+
+/// In-memory store for time slice hashes.
+///
+/// The inner store will look something like this:
+/// `[ Block_0_3, Single_3_ABC, Block_4_2, Single_6_DEF ]`
```

Comment:
```suggestion
/// `[ Block(0, 3), Single(3, [a, b, c]), Block(4, 2), Single(6, [d, e, f]) ]`
```
still took me a minute to parse this. Is this clearer to others?